### PR TITLE
Length of last word

### PR DESCRIPTION
**Pull request by nvega23**

### CODE REVIEW

File: smallest_multiple.py [hunk 1]
```diff
@@ -1,19 +0,0 @@
-# 2520 is the smallest number that can be divided by each of the numbers from 1 to 10 without any remainder.
```
Comment (kswang2400) [inline smallest_multiple.py:1]: why is this file deleted?

File: length_of_last_word.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+"""
+Given a string s consisting of some words separated
+by some number of spaces, return the length of the
+last word in the string.
+
+A word is a maximal substring consisting of
+non-space characters only.
+"""
+def last_word(word):
+	output = word.split()
```
Comment (kswang2400) [inline length_of_last_word.py:10]: your indent looks off here, make sure it's 4 spaces

File: length_of_last_word.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	return len(output[-1])
+
+if __name__ == '__main__':
+	print("running tests for last word length...")
+	assert last_word("hey there bud") == 3
```
Comment (kswang2400) [inline length_of_last_word.py:15]: nice, how about check some edge cases, 0 word strings, 1 word strings

**Review by kswang2400 — APPROVED** [review]
nice :) some notes
